app/uk/gov/hmrc/eusubsidycompliancestub/services/Store.scala

```diff
@@ -171,7 +173,7 @@ object Store {
         for {
           amendData <- amendList
           currentData <- currentList
-          bool = amendData.subsidyUsageTransactionId.get == currentData.subsidyUsageTransactionId.get
+          bool = amendData.subsidyUsageTransactionID.get == currentData.subsidyUsageTransactionID.get
```

Comment:
You can compare equality of `Option`s so the `.get` calls aren't needed.

Also `bool` could be renamed to something meaningful e.g. `hasChanges`.

Reply:
Good idea. Changes made.